.github/workflows/hourly-push-notifications.yml

```
@@ -0,0 +1,54 @@
+          current_utc_time=$(date -u +"%H:%M")
+          current_jst_hour=$(TZ=Asia/Tokyo date +"%H")
+          echo "Sending push notifications at UTC: $current_utc_time (JST: ${current_jst_hour}:00)"
+          
+          # send-push-notificationsを実行
+          response=$(curl -s -X POST '${{ secrets.SUPABASE_URL }}/functions/v1/send-push-notifications' \
+            -H 'Authorization: Bearer ${{ secrets.SUPABASE_ANON_KEY }}' \
+            -H 'Content-Type: application/json' \
+            --data '{}')
+          
+          echo "Response: $response"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)